docs/web_reference.rst

```diff
@@ -1128,6 +1135,13 @@ duplicated like one using :meth:`Application.copy`.
       :param kwargs: additional parameters for :class:`RequestHandlerFactory`
                      constructor.
 
+      .. note::
```

Comment:
I thought to do this but the parameters of `make_handler` are not described and there're just `**kwargs`.
I can add `debug` to signature and `debug` parameter description and deprecation hint but I don't think it's good idea to add deprecated parameter to signature now.

If I just use `.. deprecated :: 1.0` and the text to me it looks ugly and could be confusing.

![image](https://cloud.githubusercontent.com/assets/957767/18014685/b8ec896a-6bcc-11e6-9942-a72a02f47c8a.png)


What do you think?